### PR TITLE
Feat/add eu responsible data

### PR DESCRIPTION
**Pull request by teodorrosusn**
[AB#131444](https://dev.azure.com/plentymarkets/0af57df9-8662-4901-bb97-8e88b07ff0c9/_workitems/edit/131444)
### All changes meet the following requirements
- [ ] Changelog entry was added
- [ ] Changes have been documented
- [ ] Changes have been tested by the author
- [ ] Changes have been tested by the reviewer
- [ ] Changes to SCSS have been accounted for in plentyShop LTS Modern

@plentymarkets/plentyshop

### CODE REVIEW

File: resources/lang/de/Widget.properties [hunk 1]
```diff
@@ -1081,3 +1081,4 @@ languageDetectionRedirect = "Automatische Weiterleitung aktivieren"
 basketItemConsentLabel = "Zustimmung im Checkout"
 basketItemConsentPropertyIdLabel = "Eigenschafts-ID"
 basketItemConsentPropertyIdTooltip = "ID einer Eigenschaft eingeben. Nutzer:innen müssen Artikeln, die mit dieser Eigenschaft verknüpft sind, im Checkout über eine Checkbox zustimmen."
+manufacturerLabel = "Herstellerin"
```
Comment (ksted) [inline resources/lang/de/Widget.properties]:
```suggestion
manufacturerLabel = "Hersteller"
```

File: src/ShopBuilder/DataFieldProvider/Item/ManufacturerDataFieldProvider.php [hunk 2]
```diff
@@ -16,21 +16,53 @@
...
+        $this->addField(self::RESPONSIBLE_HOUSE_NO, "Ceres::Widget.dataFieldManufacturerResponsibleHouseNo", "item_data_field('item.manufacturer.responsibleHouseNo')");
+        $this->addField(self::RESPONSIBLE_POST_CODE, "Ceres::Widget.dataFieldManufacturerResponsiblePostCode", "item_data_field('item.manufacturer.responsiblePostCode')");
+        $this->addField(self::RESPONSIBLE_TOWN, "Ceres::Widget.dataFieldManufacturerResponsibleTown", "item_data_field('item.manufacturer.responsibleTown')");
+        $this->addField(self::RESPONSIBLE_COUNTRY, "Ceres::Widget.dataFieldManufacturerResponsibleCountry", "item_data_field('item.manufacturer.responsibleCountry')");
+        $this->addField(self::RESPONSIBLE_EMAIL, "Ceres::Widget.dataFieldManufacturerResponsibleEmail", "item_data_field('item.manufacturer.responsibleEmail')");
+        $this->addField(self::RESPONSIBLE_PHONE, "Ceres::Widget.dataFieldManufacturerResponsiblePhoneNo", "item_data_field('item.manufacturer.responsiblePhoneNo')");
+    }
+
+    public static function getFields(): array
```
Comment (ksted) [inline src/ShopBuilder/DataFieldProvider/Item/ManufacturerDataFieldProvider.php]:
```suggestion
    public static function getEuResponsibleFields(): array
```
Only `getFields` suggests to me that I get all the fields, so including the name, external name, and logo.

File: src/ShopBuilder/DataFieldProvider/Item/ManufacturerDataFieldProvider.php [hunk 3]
```diff
@@ -16,21 +16,53 @@
  */
 class ManufacturerDataFieldProvider extends DataFieldProvider
 {
-    /**
+    /** @var string */
+    public const RESPONSIBLE_NAME      = 'responsibleName';
```
Comment (ksted) [inline src/ShopBuilder/DataFieldProvider/Item/ManufacturerDataFieldProvider.php]:
```suggestion
    private const RESPONSIBLE_NAME      = 'responsibleName';
```
These don't get used anywhere else, right?

**Review by ksted — CHANGES_REQUESTED** [review]

File: src/ShopBuilder/DataFieldProvider/Item/ManufacturerDataFieldProvider.php (hunk 3 above)
Comment (ivladu-plenty) [reply src/ShopBuilder/DataFieldProvider/Item/ManufacturerDataFieldProvider.php]: You are right, I will change it to private

File: resources/lang/de/Widget.properties (hunk 1 above)
Comment (ivladu-plenty) [reply resources/lang/de/Widget.properties]: done

File: src/ShopBuilder/DataFieldProvider/Item/ManufacturerDataFieldProvider.php (hunk 2 above)
Comment (ivladu-plenty) [reply src/ShopBuilder/DataFieldProvider/Item/ManufacturerDataFieldProvider.php]: done

File: src/ShopBuilder/DataFieldProvider/Item/ManufacturerDataFieldProvider.php (hunk 3 above)
Comment (ivladu-plenty) [reply src/ShopBuilder/DataFieldProvider/Item/ManufacturerDataFieldProvider.php]: done

**Review by ksted — DISMISSED** [review]

**Review by abocsan-plenty — APPROVED** [review]

**Pull request merged**